### DISCUSSION TITLE
macOS Ventura compatibility

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [tylernguyen/x1c6-hackintosh#182] [Q&A]
So I'm looking at this GitHub repository and I do have one question. Is this EFI compatible to run macOS Ventura on the Lenovo ThinkPad X1 Carbon 6th Generation?

**Comment by dclive** [comment]
Yes, but.  See my writeup under Issues.

**Reply by buyddy** [reply]
do you have tb3 hot-plug? The ssdt working on Monterey no longer works for Ventura for me.

**Comment by rosshan-orz** [comment]
here's a different problem i've met that my thinkpad x1 carbon can't wake from sleep after i use ventura, it works perfect when i was using monterey. PLEASE HELP if anybody fixed this
